.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
+compiler: gcc
+branches:
+  only:
+    - master
+    - develop
+    - /^hotfix\/.*/
+    - /^release\/.*/
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+    #- xvfb-run -a   # initialize virtual framebuffer x-server for GUI testing
```

Comment:
this comment was supposed to be in 11th line after 'start', because it's not clear why some mysterious xvfb service must be started. I can move this comment one line higher (removing "xvfb-run -a" part) or just prune it.